# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/test/gtest/TestFileInfo.cpp
# Commit: ad7b08acadd6
# Full Hash: ad7b08acadd691fc86b52ba7f8babfaa57972c56
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-03-19 17:04:52
# Regressor Bug: 1623278
# File Overlap Count: 1
# Description:
#   Bug 1623278 - Use SafeRefPtr for FileInfo. r=dom-workers-and-storage-reviewers,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D65550
# ==============================================================================

diff -r bb094a6084b3 -r ad7b08acadd6 dom/indexedDB/test/gtest/TestFileInfo.cpp
--- a/dom/indexedDB/test/gtest/TestFileInfo.cpp	Thu Mar 19 09:32:53 2020 +0000
+++ b/dom/indexedDB/test/gtest/TestFileInfo.cpp	Thu Mar 19 09:33:31 2020 +0000
@@ -153,7 +153,7 @@
     const auto fileManager = MakeRefPtr<TestFileManager>(&stats);
     fileManager->CreateDBOnlyFileInfos();
 
-    auto fileInfos = nsTArray<RefPtr<TestFileInfo>>{};
+    auto fileInfos = nsTArray<SafeRefPtr<TestFileInfo>>{};
     for (const auto id : TestFileManager::kDBOnlyFileInfoIds) {
       fileInfos.EmplaceBack(fileManager->GetFileInfo(id));
     }
